crates/aptos-rest-client/src/lib.rs

```diff
@@ -64,13 +66,64 @@ const X_APTOS_SDK_HEADER_VALUE: &str = concat!("aptos-rust-sdk/", env!("CARGO_PK
 
 type AptosResult<T> = Result<T, RestError>;
 
+#[derive(Deserialize)]
+pub struct Table {
+    pub handle: AccountAddress,
+}
+
+#[derive(Deserialize)]
+pub struct OriginatingResource {
+    pub address_map: Table,
+}
+
 #[derive(Clone, Debug)]
 pub struct Client {
     inner: ReqwestClient,
     base_url: Url,
     version_path_base: String,
 }
 
+#[derive(Clone, Debug, serde::Serialize)]
```

Comment:
these structs duplicating `keyless/pepper/common/src/lib.rs` by design?

Reply:
I want to dedupe, but I need to deal with dependency cycles and just want to get this out.  i'll add TODO